vcsclient/github.go

```diff
@@ -989,3 +1306,31 @@ func packScanningResult(data string) (string, error) {
 type repositoryEnvironmentReviewer struct {
 	Login string `mapstructure:"login"`
 }
+
+func shouldRetryIfRateLimitExceeded(ghResponse *github.Response, requestError error) bool {
+	if ghResponse == nil || ghResponse.Response == nil {
+		return false
+	}
+
+	if ghResponse.StatusCode != http.StatusForbidden && ghResponse.StatusCode != http.StatusTooManyRequests {
+		return false
+	}
```

Comment:
Please consider this:
```suggestion
	if !slices.Contains(rateLimitRetryStatuses, ghResponse.StatusCode) {
		return false
	}
```

And declare the following file variable:
```go
var rateLimitRetryStatuses = []int{http.StatusForbidden, http.StatusTooManyRequests}
```